submissions/ses3332002/memory-game/script.js

```diff
@@ -0,0 +1,164 @@
+"use strict";
+window.onload = function() {
```

Comment:
Why you paste all code in this function?

Reply:
Well, I was taught that JS script is up to be declared in HEAD section, so to be sure that whole page is ready I use "window.onload".
I can declare JS script in bottom of BODY section, so there will be no need in "window.onload", it's just a habit.

Reply:
The script at the beginning slows down the loading of the page.
I meant that not all the code can be added to the function, but only the launch of the main functions and event handler functions.

Reply:
I remade it without "window.onload" function.